lib/serialization/booleanSpec.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+import { TypeSpec, createValidationErrorMessage } from "./typeSpec";
+import { SerializationOptions } from "./serializationOptions";
+import { SpecPath } from "./specPath";
+
+/**
+ * A type specification that describes how to validate and serialize a boolean.
+ */
+const booleanSpec: TypeSpec<boolean, boolean> = {
+  specType: "boolean",
+
+  serialize(propertyPath: SpecPath, value: boolean, options: SerializationOptions): boolean {
```

Comment:
Are you handling the parsing from string case that xml2js needs somewhere else?

Reply:
I'll do that in the SerializationPolicy.